modules/tournament/src/main/TournamentApi.scala

```diff
@@ -195,7 +195,7 @@ final class TournamentApi(
 
   def verdicts(tour: Tournament, me: Option[User], myTeamIds: Option[List[String]]): Fu[Condition.All.WithVerdicts] = me match {
```

Comment:
looks like we could accept `List[TeamId]` instead of `Option[List[String]]` since `None` is similar to `Some(Nil)` and `Nil`.